### PR TITLE
Update installation.md

### PR DESCRIPTION
**Pull request by null93**
## Purpose of this pull request

Spelling and word choice. `grep` was repeated twice in list, assumed `egrep`. Also `Inpack` was written instead of `Unpack`.

## Affected pages

https://experienceleague.adobe.com/docs/commerce-operations/tools/site-wide-analysis-tool/installation.html

- ...

## Links to Magento Open Source code

- ...

### CODE REVIEW

**Comment by jeff-matthews** [comment]
Thanks for the contribution @null93. Please complete the PR description.

**Comment by null93** [comment]
@jeff-matthews Done, apologies thought commit would speak for itself. Let me know if there is anything else. Thanks!

**Comment by jeff-matthews** [comment]
Thanks @null93! The description helps us prioritize and process contributions.

**Review by almarchenko — APPROVED** [review]

File: help/tools/site-wide-analysis-tool/installation.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Your on-premises infrastructure must meet the following requirements before inst
 
   - `nice`
 
-  - `grep`
+  - `egrep`
```
Comment (yvoronoy) [inline help/tools/site-wide-analysis-tool/installation.md]: These dependencies must be installed "php" "wget" "awk" "nice" "grep" "openssl"

**Review by yvoronoy — CHANGES_REQUESTED** [review]

File: help/tools/site-wide-analysis-tool/installation.md (hunk 1 above)
Comment (null93) [reply help/tools/site-wide-analysis-tool/installation.md]: Done, sorry for the delay. I was off for a month.

**Review by yvoronoy — APPROVED** [review]

**Comment by null93** [comment]
guess not

**Pull request closed without merging**